cyber/base/bounded_queue.h

```diff
@@ -171,14 +171,13 @@ bool BoundedQueue<T>::WaitEnqueue(const T& element) {
   while (!break_all_wait_) {
     if (Enqueue(element)) {
       return true;
-    } else {
-      if (wait_strategy_->EmptyWait()) {
+    }
+    if (wait_strategy_->EmptyWait()) {
         continue;
-      }
-      // wait timeout
-      return false;
     }
+    // wait timeout
```

Comment:
Is it indented here to remove the `return false`? The logic seems changed if "!Enqueue && !EmptyWait", in which situation old logic will `return false` immediately.

Reply:
Umm, you are right. I forgot to break while waiting timeout, so that it would return false outside the loop.